docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md

```diff
@@ -98,19 +114,90 @@ Little bit of history and context. Why is this important/useful...
+
+Genome browsers allow viewing of one type of data in the context of other data which can reveal important information about gene regulation in normal development and disease. For example, they allow researchers to ask questions about whether a genomic variant could affect expression in specific transcript variants, cell types or tissues and contribute to hypothesis development relating to genotype phenotype relationships.  Note the vertical blue line in Figure below marking the position of a disease variant and showing that the two OMIM allelic variants known for BDNF are located in the coding exon, one of these variants also overlaps with the BDNF antisense transcript.
+
+All researchers are therefore encouraged to become familiar with the use of at least one of the main browsers such as the UCSC ([UCSC Genome Browser] (https://genome.ucsc.edu/), RRID:SCR_005780), Ensembl  (Ensembl Genome Browser, RRID:SCR_013367), Epigenome browser at WashU (WashU Epigenome Browser, RRID:SCR_006208), IGV (Integrative Genomics Viewer, RRID:SCR_011793). They are designed for use by researchers without programming experience and have extensive tutorials and cases studies demonstrating the myriad of ways in which data can be loaded and interpreted to develop and support research hypothesis.
+
+* Explore features of particular chromosomal regions
+    * Investigate specific genes as well as collections of genes
+    * Search for locations of sequences and markers
+    * Retrieve annotation information for specific regions or genome-wide
+    * View your own data in context of other annotations
+    * Compare a region of one genome to genomes of other species
+
+    ### Genome Build version number
+    The Genome reference consortium
+    https://www.ncbi.nlm.nih.gov/grc
+    How does the nomenclature mean?
+    https://genome.ucsc.edu/FAQ/FAQreleases.html
+
+    For further info on Human Genome version updates I recommend you look at the updates and blog pages on the UCSC genome browser:
+    https://genome.ucsc.edu/goldenPath/newsarch.html#2019
+    http://genome.ucsc.edu/blog/patches/
 
 -------------------------------
-## Section 2: Title of section 2
+## Section 2: UCSC genome Browser
+
+In this section we will become familar with e interface of the UCSC genome browser and explore a small number of tools and datasets available.
+
+Weekly maintenance of the browser is at 5-6 pm pacific time = 11am-12pm Melbourne time. To ensure uninterrupted browser services for your research during UCSC server maintenance and power outages, bookmark a mirror site that replicates the UCSC genome browser.
```

Comment:
Do we need to be more specific whether it's AEST or ADST timezone?

Reply:
I am making more edits to this doc over the next few days for delivery as a workshop on Wednesday, it is not yet in the navigation. It is a work in progress.